src/Cake.Core/IO/Arguments/TextArgument.cs

```diff
@@ -39,5 +39,14 @@ public string RenderSafe()
         {
             return Render();
         }
+
+        /// <summary>
+        /// Returns a <see cref="string" /> that represents the current object.
+        /// </summary>
+        /// <returns>A string that represents the current object.</returns>
+        public override string ToString()
+        {
+            return this.Render();
```

Comment:
* Should call `RenderSafe` to avoid rendering secret stuff by mistake.
* Remove `this` qualifier.